the-lost-penguin/Assets/Scripts/PlayerStats.cs

```diff
@@ -71,32 +79,37 @@ public void AddExp(int exp)
             if(currentExp >= expToNextLevelList[kentoLevel - 1])
             {
                 LevelUp();
-                IncreaseStats();
-                UIController.instance.UpdateExpDisplay();
+                IncreaseStats();       
             }
         }
         //if we're at max level, don't gain anymore exp
         if(kentoLevel >= maxLevel)
-            currentExp = 0; 
+        {
+            currentExp = 0;
+            UIController.instance.UpdateExpDisplay();
+        }         
     }
 
     private void LevelUp()
     {
         //put sound effect here that tells us we leveled up
-        currentExp -= expToNextLevelList[kentoLevel];
+        currentExp -= expToNextLevelList[kentoLevel - 1];
         //update text on screen
         kentoLevel++;
-        expForNextLevel = expToNextLevelList[kentoLevel];
+        expForNextLevel = expToNextLevelList[kentoLevel - 1];
+        UIController.instance.UpdateExpLevel();
     }
 
     private void IncreaseStats()
     {
         bellySlide++; //slide distance or damage?
```

Comment:
I do not think we are doing belly slides anymore. We need to discuss this.

Reply:
alright